snowplow-tracker/src/main/java/com/snowplowanalytics/snowplow/tracker/ErrorLogging.java

```diff
@@ -0,0 +1,14 @@
+package com.snowplowanalytics.snowplow.tracker;
+
+public interface ErrorLogging {
```

Comment:
Should this be called DiagnosticLogger? I'm not sure it is clear if this is related ApplicationCrash or TrackerDianogistic logging.

Reply:
Changed to DiagnosticLogger